Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -194,6 +211,12 @@ extension Event.EventName {
     var value: String {
         switch self {
         case .OpenedPush: return "$opened_push"
+        case .OpenedAppMetric: return "Opened App"
```

Comment:
@ndurell @ajaysubra 
Thoughts on making this extension public for transparency?

Reply:
Umm sounds ok to me.

Reply:
@evan-masseau let me know once you've done this and I'll update the test app. Thanks!